src/vis_cpu/cli.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
```

Comment:
This might be better off in `./scripts` rather than in the source directory?

Reply:
That's where I originally had it, since that how HERA projects usually do it, but I actually prefer this. In this setup, I can add more commands to the cli easily as sub-commands, so I can have `viscpu profile ...` and then later add `viscpu run ...` or `viscpu inspect ...` or whatever. Happy to hear counter-arguments :-)